devfile.yaml

```diff
@@ -75,8 +60,8 @@ commands:
         type: exec
         command: |
           set -o xtrace;
-          rm -rf /usr/local/apache2/htdocs/v3/*;
-          cp -rf /projects/build/v3/* /usr/local/apache2/htdocs/v3
+          rm -rf /usr/local/apache2/htdocs/v3;
+          cp -rf ${CHE_PROJECTS_ROOT}/che-plugin-registry/output/v3 /usr/local/apache2/htdocs/
```

Comment:
Is it `/projects/che-plugin-registry/output` or `/projects/output` ?
if you change `build` to `output` you should also update the didact guide.

Reply:
also the publish to surge command

Reply:
it is `/projects/che-plugin-registry/output`

Reply:
why not `/projects/output`? it would be simpler to navigate if it is out of the git project.

Reply:
I prefer to keep everything that is generated under its parent location
Especially in case of multi root workspace

Reply:
can't we keep `/projects` as a workspace root ? `/projects/che-plugin-registry/tools/build/package.json` is not even the root folder of the git repo.

Reply:
I tried the `3. publish to surge.sh` command after adapting it to use `/projects/che-plugin-registry/output` and it fails because `tree` is no longer in the `build` container.

Reply:
I reverted the dev image change as it's not related to this PR